pkg/plugin/velero/restore_item_action.go

```diff
@@ -57,6 +57,10 @@ type RestoreItemActionExecuteOutput struct {
 	// AdditionalItems is a list of additional related items that should
 	// be restored.
 	AdditionalItems []ResourceIdentifier
+
+	// SkipRestore tells velero to stop executing further actions
```

Comment:
I would note in this comment that if this field's value is `true`, then `AdditionalItems` will be ignored.